### DISCUSSION TITLE
Open Source, my experience till now.

### DISCUSSION THREAD

**Discussion by suitupstan** [EddieHubCommunity/support#2705] [Show and tell]
What is open source? Should you open source? Why should you open source? What's the benefits? How is it like to open source?

https://krishguptadev.hashnode.dev/open-source-my-experience-till-now

**Comment by suitupstan** [comment]
- https://github.com/web3community/devprotocol.xyz by the web3community
- https://github.com/EddieHubCommunity/LinkFree by the EddieHubCommunity
- https://github.com/kaiwalyakoparkar/classroom-monitor-bot by Kaiwalya Koparkar
- https://github.com/appwrite/appwrite by Appwrite.io
- https://github.com/web3community/web3community.github.io by the web3community

**Reply by suitupstan** [reply]
Projects to contribute to as a first-timer 👆 Originally published in the article.

**Reply by AnshuPathak-88825** [reply]
thanks @krishguptadev

**Reply by aniruddh-bhandarkar** [reply]
Thanks for this. Was looking for some issues to resolve.

**Comment by SaraJaoude** [comment]
Feedback on: "What is open source? Should you open source? What's the benefits? How is it like to open source?"

I like how the blog is set out and the images make it very engaging.  Using quotes from others is also a good idea. 

For future blogs, I would suggest looking more closely at the following:

1. Consistency: the term "Open Source" is written both by capitalising the first letter of each word (**O**pen **S**ource) and not capitalised (**o**pen **s**ource).  It would be good keep this consistent throughout. 
2. Consider that writing a blog is different from speaking.  Therefore you may wish to avoid colloquialisms such as "basically" as well as instead of using "it's" writing "it is", instead of "we're" use "we are".

**Reply by suitupstan** [reply]
Thanks 🙌! I'll keep these things in mind while writing my future articles
